Example/JavaClass/template.stencil

```diff
@@ -0,0 +1,46 @@
+package {{ spec.package }};
+
+import androidx.annotation.NonNull;
```

Comment:
Do you think it makes sense to dynamically add this imports or in the template is ok?

Reply:
It would be better if it would be added dynamically.
The NonNull and Nullable imports should be added only if they are needed.
There is nothing wrong in having them there, there is no compiler error or anything like that, honestly this is just a lazy approach.